bindings/tbdex_uniffi/src/messages/cancel.rs

```diff
@@ -1,46 +1,49 @@
 use crate::errors::{Result, RustCoreError};
 use std::sync::{Arc, RwLock};
-use tbdex::messages::cancel::{Cancel as InnerCancel, CancelData};
+use tbdex::{
+    json::{FromJson, ToJson},
+    messages::cancel::{Cancel as InnerCancel, CancelData},
+};
 use web5_uniffi_wrapper::dids::bearer_did::BearerDid;
 
 pub struct Cancel(pub Arc<RwLock<InnerCancel>>);
 
 impl Cancel {
-    pub fn new(
-        bearer_did: Arc<BearerDid>,
```

Comment:
this wasn't even being signed before?

Reply:
it was, you're looking at the uniffi wrapper. so prior-to this we were only signing internal to the `tbdex` crate's `Cancel::new()` (wasn't being exposed through binding layer)